build/openresty/patches/nginx-1.21.4_06-dynamic_log_level.patch

```diff
@@ -0,0 +1,221 @@
+diff --git a/bundle/nginx-1.21.4/src/core/ngx_log.c b/bundle/nginx-1.21.4/src/core/ngx_log.c
+index eb7a989..cdf380c 100644
+--- a/bundle/nginx-1.21.4/src/core/ngx_log.c
++++ b/bundle/nginx-1.21.4/src/core/ngx_log.c
+@@ -8,6 +8,10 @@
+ #include <ngx_config.h>
+ #include <ngx_core.h>
+ 
++#if (NGX_HTTP_LUA_KONG)
++ngx_uint_t
++ngx_http_lua_kong_get_dynamic_log_level(ngx_uint_t current_log_level);
++#endif
```

Comment:
We have included `ngx_log.h` in `ngx_core.h`, so we should remove these code.

Reply:
This is a `.c` file, not a header file.

Reply:
In this patch we have declared `ngx_http_lua_kong_get_dynamic_log_level` in `ngx_log.h`, so we should not repeat it again.